Java/core/src/test/java/com/amazon/randomcutforest/CompactRandomCutForestFunctionalTest.java

```diff
@@ -866,4 +868,30 @@ public void getTotalUpdates_returnExpectedSize() {
         }
         return shingledPoint;
     }
+
+    @Test
+    public void reproduce() {
```

Comment:
Let's call this something like `testUpdateAfterRoundTrip()`.

Reply:
Done